Monika After Story/game/script-farewells.rpy

```diff
@@ -345,6 +345,51 @@ label bye_prompt_sleep:
     $ persistent._mas_greeting_type = store.mas_greetings.TYPE_SLEEP
     return 'quit'
 
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent.farewell_database,
+            eventlabel="bye_prompt_to_games",
+            unlocked=True,
+            prompt="I'll play other games.",
+            pool=True
+        ),
+        eventdb=evhand.farewell_database
+    )
+
+label bye_prompt_to_games:
+    m 1esa "Ok [player]!"
+    m 1eka "I wish I could come too and see your main character."
+    m 3rsc "But I mentioned, it depends on what you gonna play."
+    m 1eka "I couldn't stay in violance atmosphere."
+    m 1hub "Can you imagine me as putting a rifle in my hands?"
+    m 4esa "But I would love to shoot bad guys with you if we have infinite lives to try."
```

Comment:
A couple lines ago, Monika said that she would prefer not to stay in a violent atmosphere, but here you have Monika suddenly enthusiastic to kill people if she had infinite lives. This causes a conflict in not only your previous line, but also in the topic monika_othergames where she says that violent games might not be very fun for her.